Cesium3DTiles/include/Cesium3DTiles/RasterOverlayTileProvider.h

```diff
@@ -278,18 +216,26 @@ class CESIUM3DTILES_API RasterOverlayTileProvider {
    * @param outputRasterTiles The raster tiles.
    * @param outputIndex The output index.
    */
-  void mapRasterTilesToGeometryTile(
+  virtual void mapRasterTilesToGeometryTile(
       const CesiumGeospatial::GlobeRectangle& geometryRectangle,
       double targetGeometricError,
-      std::vector<RasterMappedTo3DTile>& outputRasterTiles,
-      std::optional<size_t> outputIndex = std::nullopt);
+      std::vector<std::unique_ptr<RasterMappedTo3DTile>>& outputRasterTiles,
+      std::optional<size_t> outputIndex = std::nullopt) = 0;
 
   /** @copydoc mapRasterTilesToGeometryTile */
-  void mapRasterTilesToGeometryTile(
+  virtual void mapRasterTilesToGeometryTile(
       const CesiumGeometry::Rectangle& geometryRectangle,
       double targetGeometricError,
-      std::vector<RasterMappedTo3DTile>& outputRasterTiles,
-      std::optional<size_t> outputIndex = std::nullopt);
+      std::vector<std::unique_ptr<RasterMappedTo3DTile>>& outputRasterTiles,
+      std::optional<size_t> outputIndex = std::nullopt) = 0;
```

Comment:
I don't think this one needs to be part of the `RasterOverlayTileProvider` interface, because it's never used externally.